### PR TITLE
docs: update the zksync-cli section

### PR DESCRIPTION
**Pull request by itsacoyote**
# Description

Update the ZKsync CLI section of the documentation with new pages for guides and refreshed CLI reference pages.

Moved the Quickstart and ZKsync 101 into a new section called "Start coding 🚀".

## Linked Issues

Resolves DEVRL-792

### CODE REVIEW

**Comment by jennyg0** [comment]
LGTM! imo the title in navbar would look better as Start Coding (capital C)

**Comment by itsacoyote** [comment]
> LGTM! imo the title in navbar would look better as Start Coding (capital C)

Good point, made that change 👍🏻

File: content/00.build/20.zksync-cli/90.reference/50.zksync-cli-wallet.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: zksync-cli wallet
+description: Manage your wallet on ZKsync using zksync-cli.
+---
+
+```sh
+zksync-cli wallet [sub-command] [options]
+```
```
Comment (jennyg0) [inline content/00.build/20.zksync-cli/90.reference/50.zksync-cli-wallet.md:8]: this page is missing the details

Comment (itsacoyote) [reply content/00.build/20.zksync-cli/90.reference/50.zksync-cli-wallet.md:8]: 😱

**Review by jennyg0 — APPROVED** [review]

**Comment by pcheremu** [comment]
> LGTM! imo the title in navbar would look better as Start Coding (capital C)

Hello, should we verify it first? @itsacoyote Is this PR contains all zksync-cli/Foundry template changes that we talked about?

**Comment by itsacoyote** [comment]
> > LGTM! imo the title in navbar would look better as Start Coding (capital C)
> 
> Hello, should we verify it first? @itsacoyote Is this PR contains all zksync-cli/Foundry template changes that we talked about?

It does not, this is just for the documentation of the commands. We have another PR that will have the new templates that @jennyg0 is currently working on.

**Pull request merged**